widget/settings/pages/config/bar/index.ts

```diff
@@ -72,11 +72,14 @@ export const BarSettings = () => {
 
                 Header('Media'),
                 Option({ opt: options.theme.bar.buttons.media.spacing, title: 'Inner Spacing', subtitle: 'Spacing between the icon and the label inside the buttons.', type: 'string' }),
+                Option({ opt: options.bar.media.show_artist, title: 'Show Track Artist', type: 'boolean' }),
+                Option({ opt: options.bar.media.truncation, title: 'Truncate Media Label', type: 'boolean' }),
+                Option({ opt: options.bar.media.truncation_size, title: 'Truncation Size', type: 'number' }),
```

Comment:
Should we set a minimum truncation size here?

Reply:
I intentionally didn't add one, but I will.